### PR TITLE
change backend and formalize the dir_names

### PR DESCRIPTION
**Pull request by pyangyu**
I rearranged the files and worked on the backends. I was developing the codes to make use of AmplifyDataStore and AmplifyAPI.

### CODE REVIEW

**Comment by pyangyu** [comment]
I rearranged the files and add some localization configurations, and the functionalities of the code do not change (except for some backend configurations, I tried to connect the announcement data to the backend, and probably will finish it these two days).

File: lib/LocalizedButtonResolver.dart [hunk 1]
```diff
@@ -16,6 +16,12 @@ class LocalizedButtonResolver extends ButtonResolver {
     return AppLocalizations.of(context)!.signUp;
   }
 
+  // @override
```
Comment (JavaDerek) [inline lib/LocalizedButtonResolver.dart:19]: Remove commented code

File: lib/LocalizedButtonResolver.dart [hunk 2]
```diff
@@ -16,6 +16,12 @@ class LocalizedButtonResolver extends ButtonResolver {
...
+  // String submit(BuildContext context) {
```
Comment (JavaDerek) [inline lib/LocalizedButtonResolver.dart:20]: Remove commented code

File: lib/features/announcement/controller/announcement_list_controller.dart [hunk 3]
```diff
@@ -22,6 +22,8 @@ class TripsListController {
       description: description,
       startDate: TemporalDate(DateTime.parse(startDate)),
       endDate: TemporalDate(DateTime.parse(endDate)),
+      isCompleted: false,
+      isPin: false
```
Comment (JavaDerek) [inline lib/features/announcement/controller/announcement_list_controller.dart:26]: change to "isPinned"

File: lib/features/announcement/ui/drawer_list/bottom_navigation.dart [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+import 'package:vocel/features/announcement/ui/home_page/announcement_list.dart';
+
+class AnnouncementsListPage extends StatefulWidget {
+  AnnouncementsListPage({
+    super.key,
+  }) {
+    // fetchGroups();
+  }
+
+  // Future<List> fetchGroups() async {
```
Comment (JavaDerek) [inline lib/features/announcement/ui/drawer_list/bottom_navigation.dart:23]: remove commented code

File: lib/features/announcement/ui/home_page/announcement_list.dart [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:intl/intl.dart';
+import 'package:vocel/common/utils/colors.dart' as constants;
+import 'package:vocel/data/trip_repository.dart';
+import 'package:vocel/features/announcement/controller/announcement_controller.dart';
+import 'package:vocel/features/announcement/ui/home_page/add_announcement_bottomsheet.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:vocel/models/Trip.dart';
+
+class Reminder {
```
Comment (JavaDerek) [inline lib/features/announcement/ui/home_page/announcement_list.dart:15]: Remove unused classes and break used ones into separate files, please

File: lib/features/announcement/ui/home_page/notes.dart [hunk 6]
```diff
@@ -5,6 +5,7 @@
 // import 'package:flutter/material.dart';
 // import 'package:intl/intl.dart';
 // import 'package:vocel/common/utils/colors.dart' as constants;
+// import 'package:vocel/features/announcement/ui/home_page/add_announcement_bottomsheet.dart';
```
Comment (JavaDerek) [inline lib/features/announcement/ui/home_page/notes.dart:8]: why add something commented?

File: lib/features/announcement/ui/setting_page/language_list.dart [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+                      },
+                    title: Text(Language.languageList()[index].name),
+                  )
+                ),
+              );
+            }
+        )
+
+
+        // DropdownButton<Language>(
```
Comment (JavaDerek) [inline lib/features/announcement/ui/setting_page/language_list.dart:62]: remove commented code

**Review by JavaDerek — CHANGES_REQUESTED** [review]
Please make requested changes

**Review by JavaDerek — APPROVED** [review]

**Review by erie149 — APPROVED** [review]

**Pull request merged**